core/src/main/java/com/rudderstack/android/sdk/core/Constants.java

```diff
@@ -32,6 +32,10 @@ class Constants {
     static final boolean TRACK_LIFECYCLE_EVENTS = true;
     // whether we should record screen views automatically
     static final boolean RECORD_SCREEN_VIEWS = false;
+    // default duration for session
```

Comment:
Specify time unit for this duration in comments.
In general we prefer to use "long" and Milliseconds for time duration